src/utils/emission.ts

```diff
@@ -0,0 +1,57 @@
+export const EMISSION_TABLE = [
+  { start: 0      , end: 262800 , reward: 202 },
```

Comment:
I think typical notation is [begin; end) and [first; last] where `end` is always exclusive. That would avoid confusion on whether reward for 262800 is 202 or 151

Reply:
What about 
```
{ range: [0, 262800], subsidy: 202 },
{ range: [262801, 525600], subsidy: 151 },
```

Reply:
[first; last] also good to me

Reply:
range looks good